test_runner/fixtures/parametrize.py

```diff
@@ -126,5 +126,6 @@ def pytest_runtest_makereport(*args, **kwargs):
     }.get(os.uname().machine, "UNKNOWN")
     arch = os.getenv("RUNNER_ARCH", uname_m)
     allure.dynamic.parameter("__arch", arch)
+    allure.dynamic.parameter("__lfc", os.getenv("USE_LFC") == "true")
```

Comment:
Should it be `!= "false"` for consistency?

Reply:
good catch, fixed